src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -128,8 +133,11 @@ class AuthScreens extends React.Component {
         fetchAllReports(true, true);
         fetchCountryCodeByRequestIP();
         UnreadIndicatorUpdater.listenForReportChanges();
-        getPolicySummaries();
-        getPolicyList();
+
+        if (Permissions.canUseFreePlan(this.props.betas)) {
+            getPolicySummaries();
+            getPolicyList();
```

Comment:
Curious if maybe we should fetch the policy list / policy summaries when the settings page opens so the policies will be up to date?

Reply:
Do you mean instead of or in addition to here? I did notice that if you delete a policy using the API, then the change won't appear unless E.cash refreshes. Not sure that's a problem though ... changes made to the policies in E.cash should update Onyx and trigger a refresh, right?

In other words, I'm not sure we should wait until the settings page opens to fetch the policies, but we could re-fetch them there if we want to keep E.cash and E.com more in sync?

Reply:
I was thinking only when opening the settings. But thinking more on that we'll maybe want to access the policies in the `/workspace` page as well.

>I did notice that if you delete a policy using the API, then the change won't appear unless E.cash refreshes.

Right, I don't think we care about this or subscriptions for policies yet if you meant updating in E.cash when deleting the policy via OldDot.

>changes made to the policies in E.cash should update Onyx and trigger a refresh, right?

Good question, but let's back up for a second. What changes are we talking about here? 

I think for this project it would be mainly altering policy members + creating new policies. The logic for those other two places hasn't really been built out yet so I suppose we can leave this for now and decide what to do later. I mainly suggested that we should put this in the settings page since that seems like the earliest time when we'd need to access our list of policies and not really when the app inits (as we already front load a lot of API calls).